invenio_requests/views/ui.py

```diff
@@ -53,6 +54,18 @@ def create_ui_blueprint(app):
         view_func=requests_detail,
     )
 
+    @blueprint.before_app_first_request
+    def register_menus():
+        """Register requests menu items."""
+        if app.config.get("COMMUNITIES_ENABLED", False):
```

Comment:
As discussed, please add a comment:
```suggestion
        # the config `COMMUNITIES_ENABLED` should not be used in `invenio-requests`, but
        # it was temporarly added while finizaling development to avoid yet another feature flag
        # for `invenio-requests`
        if app.config.get("COMMUNITIES_ENABLED", False):
```